platform/smart-home-app.ts

```diff
@@ -3,30 +3,46 @@
  */
 /// <reference types="@google/local-home-sdk" />
 
-import { App } from '@google/local-home-sdk';
+import { DeviceManagerStub } from './device-manager';
+import { MockLocalHomePlatform } from './mock-local-home-platform';
+import { MockNetwork } from './mock-radio';
 
-//TODO(cjdaly) fill in implementation of these methods
-class AppStub implements App {
+export class AppStub implements smarthome.App {
   private version: string;
-  private deviceManager: any;
+  public identifyHandler: smarthome.IntentFlow.IdentifyHandler;
+  public executeHandler: smarthome.IntentFlow.ExecuteHandler;
+  public reachableDevicesHandler: smarthome.IntentFlow.ReachableDevicesHandler;
+
   constructor(version: string) {
     this.version = version;
+    //  Allows Local Home Platform to access handlers
+    MockLocalHomePlatform.getInstance().setApp(this);
   }
+
   getDeviceManager(): smarthome.DeviceManager {
-    return;
+    return MockLocalHomePlatform.getInstance().getDeviceManager();
   }
+
   listen(): Promise<void> {
-    return;
+    return Promise.resolve();
   }
-  onExecute(handler: smarthome.IntentFlow.ExecuteHandler): this {
-    return;
+
+  public onExecute(executeHandler: smarthome.IntentFlow.ExecuteHandler): this {
+    this.executeHandler = executeHandler;
+    return this;
   }
-  onIdentify(handler: smarthome.IntentFlow.IdentifyHandler): this {
-    return;
+
+  public onIdentify(
+    identifyHandler: smarthome.IntentFlow.IdentifyHandler
```

Comment:
Is your linter checking the number of columns per-line? Does this go over the limit?

Reply:
I think it's currently at the default line limit for `eslint`.  I'll fix this with the dependency fixes in #30